model/src/gateway/payload/reaction_remove_emoji.rs

```diff
@@ -0,0 +1,23 @@
+use crate::id::{ChannelId, GuildId, MessageId};
+
+#[cfg_attr(
+    feature = "serde-support",
+    derive(serde::Deserialize, serde::Serialize)
+)]
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct ReactionRemoveEmoji {
+    pub channel_id: ChannelId,
+    pub message_id: MessageId,
+    pub guild_id: GuildId,
+    pub emoji: ParitalEmoji,
+}
+
+#[cfg_attr(
+    feature = "serde-support",
+    derive(serde::Deserialize, serde::Serialize)
+)]
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct ParitalEmoji {
+    id: u64,
+    emoji: String,
+}
```

Comment:
I also think we have this somewhere else, (though not completely sure)

Reply:
Nope, we don't.

Reply:
It is what we have in `/dawn/model/src/channel/reaction_type.rs` it seems to me at least

Reply:
Nope, ReacionType has Animated field that this doesn't.